Game/src/Scenes/GameplayScene.cpp

```diff
@@ -233,7 +233,7 @@ void GameplayScene::Render(double delta) {
 
 	if (m_drawLN && std::get<9>(scores) > 0) { // Same Animation like DrawCombo
 		m_amplitude = 5.0;
-		m_wiggleTime = 24.0 * m_lnTimer;
+		m_wiggleTime = 25.0 * m_lnTimer;
```

Comment:
you can adjust m_wiggleTime from 24 to 30 (or maybe on new engine it should be good at 30) otherwise animation will became weird